contents/product-engineers/best-customer-retention-metrics.md

```diff
@@ -0,0 +1,246 @@
+
+It’s not just customers who churn, revenue churns with them. This metric helps you understand how much money your business makes from each user over time. It can help you spot where you’re losing revenue due to downgrades, people failing to renew, or, worst of all, cancellations. 
+
+While your customer churn rate looks at the number of customers you lose in a given period, your revenue churn specifically focuses on the dollar value of recurring period lost, or gained, in a certain period.
+
+### How is it useful?
+
+Any subscription-based business should be measuring their revenue churn rate. These days, that’s pretty much all businesses from coffee subscription businesses to B2B SaaS companies and everything in between. Businesses that sell big, high-value items (think houses, industrial equipment) likely don’t need to focus on revenue churn.
+
+### How to calculate revenue churn rate
+
+You can calculate your revenue churn rate based on either gross or net revenue. A focus on gross revenue is simpler as it’ll show you how much revenue is churning out of your business, but a focus on net revenue will show both the amount of revenue lost from, say, customers downgrading, but also the revenue gained from customer upgrading.
+
+Let’s do the math. For your gross revenue churn rate, take your churned revenue in a month and divide it by the monthly recurring revenue (MRR) from the previous month, and multiply that total by 100. 
+
+For example, Hogflix, an imaginary hedgehog-themed Netflix competitor, had an MRR of $100,000 in May. In April, its churned revenue was $17,000. Divide $17,000 by $100,000 to get 0.17, then multiply that by 100. Voila, Hogflix has a gross revenue churn rate of 17%.
+
+To work out your net revenue churn, find your churned revenue in a period and subtract your expansion revenue and divide that total by your monthly recurring revenue in the previous period. Now multiply that new total by 100.
+
+For example, Hogflix had an MRR of $100,000 in May. In April, its churned revenue was $17,000 and its expansion revenue was $3,000. So subtract $3,000 from $17,000 to get $15,000 and divide that by $100,000 to get 0.15. Multiply that by 100 to get 15%.
+
+### What’s a good revenue churn rate?
+
+This depends again on how much your customers pay and how often they pay. If your business is based on monthly subscriptions, and your average revenue per account is $10 or less per month, then aim for monthly net revenue churn of around 6% or less. 
+
+The more customers pay, the lower your churn should be. At $250 average revenue per account per month, aim for a net revenue churn [of 1.4% or less](https://techcrunch.com/2022/04/20/study-up-on-churn-rate-basics-to-set-customer-and-revenue-benchmarks/), according to data compiled by analytics platform ChartMogul. This sort of churn rate also assumes that your customers are paying annually and, therefore, churn less often because they can’t ditch their contract.
```

Comment:
I think this whole section is missing some important context. Net revenue retention can be negative, right? And that would be a very good thing for a SaaS business, which is our main audience, so we should talk about that a bit more.

As an additional, I wonder if we should have something about net / gross revenue retention. Afaik, they're basically the same thing but express slightly differently. I know we use NRR specifically. Good bit on it here: https://www.lightercapital.com/blog/net-revenue-retention-vs-gross-revenue-retention